packages/components/src/font-size-picker/font-size-picker-toggle-group.tsx

```diff
@@ -29,7 +30,12 @@ const FontSizePickerToggleGroup = ( props: FontSizePickerToggleGroupProps ) => {
 				<ToggleGroupControlOption
 					key={ fontSize.slug }
 					value={ fontSize.size }
-					label={ T_SHIRT_ABBREVIATIONS[ index ] }
+					label={
+						fontSize.shortName &&
+						hasValidShortName( fontSize.shortName )
+							? fontSize.shortName
+							: T_SHIRT_ABBREVIATIONS[ index ]
```

Comment:
Could probably rename this to be `DEFAULT_SHORT_NAMES` or something if we wanted to consistently refer to it as a "short name".

Reply:
Done in [6b32b74](https://github.com/WordPress/gutenberg/pull/46586/commits/6b32b7409d4e6089ac121a14f222fad853796b06)